src/utils/grammars/lexedMichelson.js

```diff
@@ -0,0 +1,187 @@
+// Generated automatically by nearley, version 2.16.0
```

Comment:
If this is generated code, should there be a build step to create it rather than have in the repo?

Reply:
Yes, this is an issue that I will work on correcting.

Reply:
The parser will not need compilation on every build of conseiljs, only when the parser changes, which will happen when we add enhancements or if, for some reason, the Michelson spec changes. I added an npm command "compile-michelson-parser", just in case we need this.